setup.cfg

```diff
@@ -44,6 +44,10 @@ install_requires =
     click
     websockets~=13.1
 
+[project.optional-dependencies]
```

Comment:
I'm not sure how you would install this. I tried it with `pip install -e .[ws]` and that didn't work. But it does work if you move the *ws* section to `[options.extras_require]` ??

Reply:
Thank you, I see the issue. I'll follow [this PEP 621 recommendation](https://peps.python.org/pep-0621/#other-names-for-dependencies-optional-dependencies) to normalize the name of the config setting.

The bug was hidden, because I had forgotten to remove `websockets` from the regular `install_requires` setting.